### PR TITLE
Links act in the same way and use the same colors

### PR DESCRIPTION
**Pull request by hemanthsavasere**
Hi @hemantjadon,
Did the changes as requested.
Also I changed a single line as the conflict was there in clean-min.blog.css.

image
The selected area is where changes has been made.

### CODE REVIEW

File: css/clean-blog.min.css [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
...e,opacity .3s ease;-moz-transition:top .3s ease,opacity .3s ease;-ms-transition:top .3s ease,opacity .3s ease;transition:top .3s ease,opacity .3s ease}.floating-label-form-group::not(:first-child){padding-left:14px;border-left:1px solid #eee}.floating-label-form-group-with-value label{top:0;opacity:1}.floating-label-form-group-with-focus label{color:#e12b00}form .row:first-child .floating-label-form-group{border-top:1px solid #eee}.btn{font-size:14px;border-radius:0;padding:15px 25px}.btn-lg{font-size:16px;padding:25px 35px}.btn-default:focus,.btn-default:hover{background-color:#e12b00;border:1px solid #e12b00;color:#fff}.pager{margin:20px 0 0}.pager li>a,.pager li>span{font-size:14px;padding:15px 25px;background-color:#fff;border-radius:0}.pager li>a:focus,.pager li>a:hover{color:#fff;background-color:#e12b00;border:1px solid #e12b00}.pager .disabled>a,.pager .disabled>a:focus,.pager .disabled>a:hover,.pager .disabled>span{color:#777;background-color:#333;cursor:not-allowed}::-moz-selection{color:#fff;text-shadow:none;background:#e12b00}::selection{color:#fff;text-shadow:none;background:#e12b00}img::selection{color:#fff;background:0 0}img::-moz-selection{color:#fff;background:0 0}
```
Comment (hemantjadon) [inline css/clean-blog.min.css:5]: Why changes in the this file?

**Review by hemantjadon — CHANGES_REQUESTED** [review]
There should not be changes in `clean-blog.min.css` I guess. This is the vendor file.

**Comment by hemantjadon** [comment]
@hemanthsavasere Why you close your PR's always??

**Comment by hemanthsavasere** [comment]
Hi @hemantjadon 
Sorry, I didin't mean to offend anyone.
The problem here is,
the custom.css file's styles are applying to some, but I cannot override other links whose properties already has been defined in css/clean-blog.min.css for that particular elements in the paragraph.
I cannot override these.
Example: In Line 5 of the clean-blog.min.css
p a{text-decoration:underline}a:focus,a:hover{color:#e12b00}
is already specified for paragraph elements and I cannot override with in default way I need to use a class or an ID or override it in other way.
Thanks

**Pull request closed without merging**